src/Bundler.js

```diff
@@ -473,6 +473,7 @@ class Bundler extends EventEmitter {
   }
 
   middleware() {
+    this.bundle();
```

Comment:
You should probably also remove this.bundle() from the server function as this would cause bundle to be called twice on built-in server

Reply:
@DeMoorJasper Hmm, I think that `Server.serve` calls `Server.middleware` and not `Bundler.middleware`. Therefore bundle is never called twice.

Reply:
@DeMoorJasper Plus the `serve` in `Server.middleware()` just uses the `serve-static` pkg. :)

Reply:
Ow damn sorry for the mixup, looks good than, kinda weird that one of the tests is failing though

Reply:
Indeed, considering that it's not related to the changes:
```
  99 passing (33s)
  1 failing
  1) css should produce two bundles when importing a CSS file:
     evalmachine.<anonymous>:87
},{"./local":9,"./index.css":14}]},{},[5]) module == 'object' && module &&
                                           ^^^^^^
SyntaxError: Unexpected identifier
      at createScript (vm.js:80:10)
      at Object.runInContext (vm.js:119:10)
      at run (test\utils.js:90:6)
      at Context.<anonymous> (test\css.js:24:18)
      at <anonymous>
```